reference/build_helpers/cmake.rst

```diff
@@ -292,3 +292,42 @@ Environment variables
 
 There are some environment variables that will also affect the ``CMake()`` helper class. Check them in the
 :ref:`CMAKE RELATED VARIABLES<cmake_related_variables>` section.
+
+Example
+-------
+The following example of ``conanfile.py`` shows you how to manage a project with conan and CMake.
+
+.. code-block:: python
+
+    import conans
+    import conans.tools
+
+    class SomePackage(conans.ConanFile):
+        name = "SomePackage"
+        version = "1.0.0"
+        settings = "os", "compiler", "build_type", "arch"
+        options = {"shared": [True, False]}
```

Comment:
Remove not relevant information for this example: options & default_options, keep_imports, requires